### ISSUE TITLE
libretro core crashes when trying to save state

### ISSUE THREAD

**Issue by gingerbeardman** [AZO234/NP2kai#165]
- i don't mind if save states are not possible
- but it would be nice if it didn't crash

**Comment by DSwizzy** [comment]
mines did the same thing too just now on my Nintendo Switch... i wish this dude made a standalone version of this emulator instead....

**Comment by Mihara** [comment]
Happens for me on Android with the freshly updated core, now...

**Comment by tharubberchicken** [comment]
Yes, unfortunately for me too. It has been a while since I first came across the issue and it still occurs. I have also heard that if you make your own build on the device you are using it on then the issue is corrected. I don't build my own, but maybe I'll have to try...

**Comment by tharubberchicken** [comment]
Found the solution! The fix is crazy and seemingly not so obviously related to the problem, at least to me. What you need to do is enable _Mixer_ under _Audio_ --> _Menu Sounds_ in the main settings. Ensure you do save your settings and also your per game/core/folder settings (if you use them) otherwise the setting will revert back upon loading a game. Also fixes similar save issues with Mupen64Plus-Next using the GLideN64 plugin.

**Comment by skyksit** [comment]
I have the same problem. save state does not work
